docs/tutorial.md

```diff
@@ -878,6 +880,51 @@ local function do_something(p: MyPoint)
    -- ...
 end
 ```
+## Integers and numbers
+
+The integer sub-type was added with release 0.13.0. It is of undefined
```

Comment:
The tutorial shouldn't refer to Teal versions, it should refer only to the current version, assuming a newcomer doesn't know or care about past Teal history.